src/App.tsx

```diff
@@ -1,15 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(
+    (movie) => {
+      if (movie.title.toLowerCase().includes(query.toLowerCase())
+        || (movie.description.toLowerCase().includes(query.toLowerCase()))) {
```

Comment:
Move it to a separate method

Reply:
I don't think it will be better solution. Now there are just a 2 lines and this variant is very clear for understanding the code.
I would like insist on this variant. Please approve it.

Reply:
Shorter is not always better.
Sometimes it is better to write extra code but make it as clear as possible, especially if you are working on a real project with a team of developers.